### PR TITLE
docs: add a listing of supported input formats

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,15 @@ Here is an overview of the architecture of GUAC:
 
 ![image](https://user-images.githubusercontent.com/3060102/182689908-477f4770-1142-4c18-8fa9-16d93dcf84b4.png)
 
+## Supported input formats
```
Comment (fridex) [inline README.md:36]: to reviewer: please let me know if versions should be considered in the listing

File: README.md [hunk 2]
```diff
@@ -33,6 +33,15 @@ Here is an overview of the architecture of GUAC:
...
+
+- [CycloneDX](https://github.com/CycloneDX/specification)
+- [Dead Simple Signing Envelope](https://github.com/secure-systems-lab/dsse)
+- [OSV](https://ossf.github.io/osv-schema/)
```
Comment (lumjjb) [inline README.md:40]: We. support OSV by proxy of certification attestations. So maybe minus this and then add [Intoto ITE6](https://github.com/in-toto/attestation) as another input format.. @pxp928  can confirm

**Review by lumjjb — COMMENTED** [review]
Hi @fridex! Thanks for opening up this PR! Just 1 minor comment and also - would you be able to fix the DCO?  (help with DCO here: https://github.com/guacsec/guac/pull/257/checks?check_run_id=9890226728)

File: README.md (hunk 2 above)
Comment (fridex) [reply README.md]: Thank you. Adjusted based on your comments. PTAL.

**Review by mihaimaruseac — APPROVED** [review]
Thank you!

**Review by pxp928 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**